dandiapi/api/__init__.py

```diff
@@ -0,0 +1,2 @@
+# TODO remove this after migration is complete
```

Comment:
Could you file an issue for this and mark it with `migration` or whatever label may already exist? That will make it easier to get a complete list of tasks together in a formal migration plan.